docs/src/widgets/wearos.md

```
@@ -2,6 +2,8 @@
 
 Perry widgets can compile to Wear OS Tiles using `--target wearos-tile`. Tiles are glanceable surfaces in the Wear OS tile carousel and watch face complications.
 
+> For full **Wear OS apps** (not glanceable Tiles), see [Wear OS](../platforms/wearos.md).
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)